lib/index.js

```diff
@@ -57,7 +57,11 @@ class PuppeteerPlugin {
 				await page.close();
 
 				// convert utf-8 -> binary string because website-scraper needs binary
-				return Buffer.from(content).toString('binary');
+				// https://github.com/website-scraper/node-website-scraper?tab=readme-ov-file#afterresponse: (binary) This is advised against because of the binary assumption being made can foul up saving of utf8 responses to the filesystem.
+				return {
+					body: content,
+					encoding: 'utf8'
```

Comment:
Should we do the same we did in [website-scraper](https://github.com/website-scraper/node-website-scraper/blob/bbf0101c427d156b4ada0f3f6ad33ce82c0b2488/lib/request.js#L13-L16)?

Reply:
Hi @azlarsin,
thank you for your contribution 👍  and sorry for the very late response from my side.

Hi @aivus,
thank you for pinging me. I think, it's a good idea to use the same approach, that already exists in the main module. It would also be nice to have some tests for it (we can use an example in the PR comment).